Add docs for project-settings.json (#305)
vaadin/docs: articles/tools/designer/using-designer/faq.asciidoc

 * Modify or add the `-vm` parameter as instructed in the Eclipse wiki: https://wiki.eclipse.org/Eclipse.ini#Specifying_the_JVM

Please check https://github.com/vaadin/designer/blob/master/RELEASE-NOTES.md#requirements for more details on supported versions.

=== Does Vaadin Designer support Maven multi-modules project structures?

> Yes. A maven multi-modules project can also have different Vaadin versions (from Vaadin 8 to the latest version) in seperate sub-modules.

Reviewer: ```suggestion
Yes.
A Maven multi-module project can also have different Vaadin versions in separate sub-modules.
```

> In case a module (A) want to re-use the same `node_modules` dependencies from another module (B), you can create `project-root/(module_A]).vaadin/designer/project-settings.json` to define `node_modules` location.

Reviewer: ```suggestion
In case a module (A) want to re-use the same `node_modules` dependencies from another module (B), you can create `project-root/[module_A]/.vaadin/designer/project-settings.json` to define `node_modules` location.
```

[source,json]
----
{

>   "--node.modules.project.name": "(module B's name)"

Reviewer: ```suggestion
  "--node.modules.project.name": "[module_B]"
```
